spring-cloud-deployer-kubernetes/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -90,6 +90,20 @@ public static class LimitsResources {
          */
         private String memory;
 
+        /**
+         * Container resource ephemeral-storage limit.
+         */
+        private String ephemeralStorage;
+
+        /**
```

Comment:
Did you choose `size` of `2Mi` and `1Gi` because they are common choices? I can envision the next ticket comes in and says "I want to set 'size' in 'hugepages-<size>'". 

Also, I believe this is a linux only feature. 

 common choices for `hugepages-N`?

Reply:
Yes. hugepages is linux only. Most containers run in linux hosts. It seems that 2Mi and 1Gi are the only 2 options supported.